impeller/renderer/backend/vulkan/allocator_vk.h

```diff
@@ -28,8 +29,8 @@ class AllocatorVK final : public Allocator {
   static constexpr size_t kPoolCount = 3;
 
   fml::RefPtr<vulkan::VulkanProcTable> vk_;
-  VmaAllocator allocator_ = {};
-  VmaPool staging_buffer_pools_[kPoolCount] = {};
+  UniqueAllocatorVMA allocator_;
```

Comment:
So, to state the obvious as I'm still a C++ noob:

The RAII wrappers handle destroying the native resources, and we always go though destructors in reverse order. So staging_buffer_pools_ being declared after allocator_ ensures that we destroy the pools with the allocator before destroying the allocator itself?

Reply:
Right.